src/cueUtilities.js

```diff
@@ -249,28 +254,21 @@ module.exports = function (params, cy, api) {
         cy.on('mouseout', 'node', data.eMouseOut = function(e) {
           let node = this;
           if (isMouseHoveringOver(node)) {
-            hoveredGroup = null;
+            // note: hoveredGroup is not set to null if we go onto whiteboard, probably need to change mouseover and remove 'node'
+            // currently is not a problem as leftover hoveredGroup won't be used and will be reset
```

Comment:
Not sure what the above comment means. Maybe outdated? Anyways, take a look and see if it's still pertinent.